### DISCUSSION TITLE
how to see tiny V (volt) measurement

### DISCUSSION THREAD

**Discussion by hatsuneelectronics** [fhdm-dev/scoppy#149] [Q&A]
hi. greetings my friend. i need some perspective of view about my razberry pie pico and scoppy assembled ocilloscope.
https://www.youtube.com/watch?v=KUzuuq-3Tc8. this man is speaking japanese so maybe you can't understand but here is Schematic
also after part of video show us,1/10 probe Schematic(11:13). if build t
his like in this video,ocilloscope can bear untill 33v but parameter will be 1/10 ( for exsample if you detect 10v in some machine,then this pico count like 1.0v)
so what i want to say in point, i can't measure tiny volt :/.
in scoppy when using horizontal time/div and used - botton, easy to see wave even micro electoronics wave
but when i do, i coudn't see tiny volt like 0.5 volt( i mean in this case 5 volt measurement)
does someone know how to see tiny volt on scoppy?
![スクリーンショット 2024-07-20 070824](https://github.com/user-attachments/assets/da2928cb-88c4-4bc2-8f8f-d38ceb34e3bf)
![IMG_20240720_073255](https://github.com/user-attachments/assets/52d3324a-f6eb-45f1-8d7c-04016497694b)

**Comment by Jason-BCIT** [comment]
The Pico has a 12 bit Analog Digital Convertor (ADC), allowing it to measure 4096 voltage levels.

If you use a voltage divider and offset to allow Scoppy to read +-30V, then each reading will be in 60V/4096 = 0.015V increments

But if you use a voltage divider to read +-3V then each increment will be 0.0015V 

Remember that the Pico is a general purpose microcontroller… it might be capable of measuring 4096 levels of input, but sometimes a few of those voltage levels seem to blend together. It might not be as precise as a dedicated ADC chip.

This is the same on any scope or meter… you have to select the correct voltage range for the signal you want to read.

Scoppy is very cool because it has voltage range pins that can either detect the position of a physical switch (if you use a physical switch to select the voltage range) or to control a semiconductor IC switch if you want to control the voltage range from the app.

I hope that is a useful reply. I may not have fully understood what you wanted to know.

**Reply by Jason-BCIT** [reply]
Oh, yes… one more thing! When using voltage dividers you also need to set the voltage range in the Scoppy app.

Jason
